src/lib/features/feature-lifecycle/feature-lifecycle-store.ts

```diff
@@ -10,6 +10,8 @@ import type { StageName } from '../../types';
 type DBType = {
     stage: StageName;
     created_at: string;
+    status?: string;
```

Comment:
normally we have a rule of thumb that we don't add anything in migration prs but I think the optional one should be fine.